mautic/mautic
# Fix email queue resend process

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
I've noticed that Mautic never attempts to resend email through queue and sets DNC flag after first try (part of the code was unreachable). This PR resolves this issue.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Enable email queue
3. Send failing email (e.g. to server that does not respond)
4. Email queue command should attempt to resend email few times before removing it and setting DNC flag on contact
